docs/fetching-alert-contacts-from-uptime-robot.md

```diff
@@ -0,0 +1,31 @@
+# Fetching alert contacts from UpTime Robot
```

Comment:
Add a link to this file in the main readme under heading of configuring providers and then uptime robot or something